packages/client/src/glsp-sprotty/index.ts

```diff
@@ -342,3 +317,31 @@ import viewportModule from 'sprotty/lib/features/viewport/di.config';
 import zorderModule from 'sprotty/lib/features/zorder/di.config';
 import graphModule from 'sprotty/lib/graph/di.config';
 import modelSourceModule from 'sprotty/lib/model-source/di.config';
+
+export {
+    buttonModule,
+    defaultModule,
+    edgeEditModule,
+    edgeIntersectionModule,
+    edgeLayoutModule,
+    expandModule,
+    fadeModule,
+    viewportModule as glspViewportModule,
```

Comment:
```suggestion
    viewportModule as sprottyViewportModule,
```

Shouldn't this be prefixed with `sprotty`?